Header header
geometry_msgs/Pose pose
geometry_msgs/Pose poseHat
float32 pRatio
int16 keyInd
